### DISCUSSION TITLE
Minor improvement: Keep Extra networks view on refresh

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#1114] [Ideas]
When the user is in a subfolder in their Extra networks (such as LORA) and an user-initiated event causes the list to refresh and **_there is a change_** (such as clicking on the button to replace extra network preview image or clicking the refresh button to reload the directory after deleting a LORA), the UI does not remember the position the user was in, reverting to the standard view from the root folder.

Could the position be remembered so that after refresh, the user stays on the same subfolder?

**Comment by vladmandic** [comment]
sounds easy, right. perhaps in the new ui which is under development, but in the current one nearly impossible as extra networks page is completely rebuild on each change so there is no concept of "last position".

**Comment by mark-wd** [comment]
It's as I feared. 😂
